leapcode/bitmask_client
# Make leap a namespace package and split leap.common as external submodule

Note that, for now, I've moved the leap.common submodule to https://github.com/kalikaneko/leap_common

To test this properly:
1. read the diff, comment there.
2. get a fresh virtualenv for testing leap_client
3. run setup.py develop (should install regular deps and symlink your source tree)
4. run pip install -r pkg/requirements (should install leap.common external package from git)

Once we estabilize this, we should move leap_common to leap repo and push to pypi.
